chore: prepare for launch (#4728)
iotaledger/firefly: README.md

## About

Firefly Shimmer is the official wallet for managing your tokens and assets on the Shimmer network.

## Installation

There are a few options to install Firefly Shimmer:

-   Download from [official Firefly Shimmer website](https://firefly.iota.org)
-   Download from the latest [GitHub release](https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0)
-   Build from source code (see our handbook guide for ["Environment Setup"](https://iotaledger.github.io/firefly/guides/environment-setup/))

## Contributing

Help out the Firefly project by filing a [bug report](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug&template=bug_report.yml), making a [feature request](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=feat&template=feature_request.md) or opening a [pull request](https://github.com/iotaledger/firefly/pulls/).

> If making a code contribution, be sure to checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) beforehand.

Reviewer: ```suggestion
Checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) before making a code contribution.
```

Author: Suggest these to matt, this isn't even visible unless in the branch.

> For all of the details, see our [development handbook](https://iotaledger.github.io/firefly/).

Reviewer: ```suggestion
See our [development handbook](https://iotaledger.github.io/firefly/) for more details.
```

Author: Suggest these to matt, this isn't even visible unless in the branch.

Reviewer: I pushed the changes
